src/System.Runtime.Extensions/src/System/HashCode.cs

```diff
@@ -0,0 +1,81 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace System
+{
+    public struct HashCode : IEquatable<HashCode>
+    {
+    	private readonly int _hash;
+        
+        private HashCode(int hash)
+        {
+            _hash = hash;
+        }
+
+        public static HashCode Empty => default(HashCode);
+    
+        public static HashCode Create(int hash) => new HashCode(hash);
+        
+        public static HashCode Create<T>(T value)
+        {
+        	return Create(GetHashCodeDefaultComparer(value));
```

Comment:
Eh, no, not really. The body of `Create` seems trivial enough and unlikely to change. I'll change these to use the ctor.